translations/ar/LC_MESSAGES/messages.po

```diff
@@ -1,18 +1,13 @@
 # Arabic translations for PROJECT.
-# Copyright (C) 2022 ORGANIZATION
 # This file is distributed under the same license as the PROJECT project.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
 #
 #, fuzzy
```

Comment:
All looks good to me! Do you know why this is marked as `fuzzy`? In my understanding this keyword was only used for non-translated "guessed" translations by gettext.

Reply:
I think to indicate this file can contain this keyword, or the file has changed. All po files have this here.